Hymns/Display/DisplayHymnViewModel.swift

```diff
@@ -10,6 +10,7 @@ class DisplayHymnViewModel: ObservableObject {
 
     @Published var isLoaded = false
     @Published var title: String = ""
+    @Published var searchTitle: String = ""
```

Comment:
Can you just use `computedTitle`

Reply:
ComputedTitle is like Hymn 608. So we wouldn't want to use that for what Sound cloud searches or we will never find what we are looking for

Reply:
I think computed title is Hymn 608: TITLE_OF_HYMN. Can you try it out to see how good soundcloud's search algorithm is? I thought it'd be even better because it searches both the title of the hymn and Hymn 608 so we get both results. But that might not be true